### PR TITLE
v2v: add skip_vm_check for convert_vm_to_ovirt

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.esx.vm.7_0.linux.latest6.arch_x86_64.raw_f.NFS.rhv_upload.rhv_direct.rhv_noverifypeer.nopreallocated.it_vddk: PASS (657.74 s)

**Comment by xiaodwan** [comment]
@chloerh @yantian0 @chunfuwen Please help to review this PR, thanks.

**Review by yantian0 — APPROVED** [review]

File: v2v/tests/src/convert_vm_to_ovirt.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def run(test, params, env):
     ovirt_hostname = params.get("ovirt_engine_url").split('/')[2]
     ovirt_ca_file_path = params.get("ovirt_ca_file_path")
     local_ca_file_path = params.get("local_ca_file_path")
+    skip_vm_check = params.get('skip_vm_check', 'no')
```
Comment (chloerh) [inline v2v/tests/src/convert_vm_to_ovirt.py:61]: I didn't see adding `skip_vm_check` to any case, and it seems to be `no` by default. Am I missing something here? Or maybe we are planning to add cases later, which makes sense.

Comment (xiaodwan) [reply v2v/tests/src/convert_vm_to_ovirt.py:61]: Yes, It's no by default and it's a reserved variant for gating on OSP. We may need to replace it to 'yes' when rhv engine is on OSP is used. libvirt-ci will replace the default value to 'yes'.

Comment (chloerh) [reply v2v/tests/src/convert_vm_to_ovirt.py:61]: Of course. Thank you for explaining so clearly. :+1:

**Review by chloerh — APPROVED** [review]

**Pull request merged**